ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/WithdrawalsValidator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.hyperledger.besu.ethereum.api.jsonrpc.internal.methods.engine;
+
+import org.hyperledger.besu.ethereum.core.Withdrawal;
+import org.hyperledger.besu.ethereum.mainnet.TimestampSchedule;
+
+import java.util.List;
+import java.util.Optional;
+
+public class WithdrawalsValidator {
+
+  static boolean isWithdrawalsValid(
+      final TimestampSchedule timestampSchedule,
+      final long timestamp,
```

Comment:
nit: think this can also be renamed to a more significant context seems like it's always coming from a newPayload so payloadTimestamp or newPayloadTimestamp?

Reply:
Have gone with newPayloadTimestamp